src/components/styled.components/Input.styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export const Input = styled.input`
+	margin-bottom: 20px;
+	margin-left: 100px;
+	padding: 5px;
+	border: 3px solid white;
+	border-radius: 20px;
+	color: #2d6cde;
+	font-family: cursive;
+`;
+
+export const Input_done = styled.input`
```

Comment:
to be less repetitive, you can use your Input component as base to style the `Input_done` component:

```css
export const Input_done = styled(Input)`
  margin-bottom: 0;
  margin-top: 10px;
```

Then you don't need to redeclare your stylings and you only need to adjust the declarations that are new or not necessary anymore.